src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/CreateSDOCallbackHandler.java

```diff
@@ -1598,6 +1599,8 @@ private CallbackResponse submitSDO(CallbackParams callbackParams) {
                 dataBuilder.hmcEaCourtLocation(featureToggleService.isLocationWhiteListedForCaseProgression(
                     caseData.getCaseManagementLocation().getBaseLocation()) ? YES : NO);
             }
+        } else if (isLipCaseWithProgressionEnabledAndCourtWhiteListed(caseData)) {
```

Comment:
I think the first **if** condition will still run once our feature toggle is set to ON.
Check the method on 1590  **sdoSubmittedPreCPForLiPCase**, we might have to remove it to instead check:
**If** part of national rollout && not LIP
**Else if** isLipCaseWithProgressionEnabledAndCourtWhitelisted
**Else** default to eaCourt NO.

Reply:
Done